src/main/java/xyz/nucleoid/plasmid/fake/FakeItem.java

```diff
@@ -16,26 +15,20 @@
      * @param stack The original stack
      * @return The stack to send to the client
      */
-    @Deprecated
     default ItemStack asProxy(ItemStack stack) {
-        ItemStack proxy = new ItemStack(this.asProxy(), stack.getCount());
-
-        CompoundTag tag = stack.getTag();
-        if (tag != null) {
-            proxy.setTag(tag.copy());
-        }
-
-        return proxy;
-    }
-
-    static Item getProxy(Item item) {
-        if (item instanceof FakeItem) {
-            return ((FakeItem) item).asProxy();
-        }
-        return item;
+        return stack;
     }
 
-    @Deprecated
+    /**
+     * Resolves the proxy {@link ItemStack} instance to be sent to the client. If the associated {@link Item} of this
+     * {@link ItemStack} is an instance of {@link FakeItem}, the {@link #asProxy(ItemStack)} method will be used
+     *
+     * <p>This method is intended for use in case the item has information which the client would not be able to
+     * normally see, so that lore could be appended, or its visuals changed
+     *
+     * @param stack The server side {@link ItemStack}
+     * @return The {@link ItemStack} to send to the client
+     */
     static ItemStack getProxy(ItemStack stack) {
```

Comment:
I believe there were issues when proxying ItemStacks with Server-Translations- is this tested?

Reply:
They don't seem to work; I'll try to fix this now

Reply:
I believe the reason for this not working is that Server-Translations depends on the ItemStack's Item being the source fake item when it checks to translate the name. 
https://github.com/arthurbambou/Server-Translations/blob/abc6e0c4594fa2a8bbb195cfff7062678da5a86c/api/src/main/java/fr/catcore/server/translations/api/mixin/packet/PacketByteBufMixin.java#L50-L57